packages/rule-components/src/ReportDetails/index.scss

```diff
@@ -1,13 +1,28 @@
 .ins-c-rule__report-detail {
-  padding: var(--pf-global--gutter--md) var(--pf-global--gutter);
+  padding: var(--pf-global--gutter--md) 40px;
 
   .ins-c-rule__report-detail-link {
     margin-bottom: var(--pf-global--spacer--xs);
     margin-left: var(--pf-global--spacer--form-element);
   }
 
-  .ins-c-rule__risk-detail {
+  .ins-c-rule__risk-detail-title {
     margin-bottom: var(--pf-global--spacer--sm);
+    font-weight: 200;
+  }
+
+  .ins-c-rule__risk-detail-description-text {
+    color: #737679;
+    size: 14px;
+  }
+
+  .ins-c-rule__link-in-description {
+    display: block;
+    margin-top: var(--pf-l-stack--m-gutter--MarginBottom);
   }
 }
 
+.ins-risk-of-change-icon {
```

Comment:
Where is this class used? Also, can you move this into `ins-c-rule__report-detail` rule?

Reply:
> Where is this class used?

in RiskOfChangeIcon.js